cosmos_rl/policy/model/deepseek_v3/configs/deepseek_config_small.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
One unit test that would be interesting to add here is a small overfit test for the small model. E.g. just train on the same example over and over and check that loss is going to zero. Hopefully this could run in the CI and give us peace of mind.

Reply:
Added the unit test, though the test setup isn't ideal. Please see the added test.